core/modules/loader/CentralClient.cc

```diff
@@ -65,45 +65,45 @@ void CentralClient::start() {
 }
 
 
-void CentralClient::handleKeyInfo(LoaderMsg const& inMsg, BufferUdp::Ptr const& data) {
-    LOGS(_log, LOG_LVL_DEBUG, "CentralClient::handleKeyInfo");
+void CentralClient::handleKeyLook(LoaderMsg const& inMsg, BufferUdp::Ptr const& data) {
+    LOGS(_log, LOG_LVL_DEBUG, "CentralClient::handleKeyLook");
 
     StringElement::Ptr sData = std::dynamic_pointer_cast<StringElement>(MsgElement::retrieve(*data));
```

Comment:
`auto` would work here better. Besides, variables which are initialized at a point of their declarations and which are never changed later might be `const`, just in case:
```
auto const sData = std::dynamic_pointer_cast<StringElement>(MsgElement::retrieve(*data));
```